istioctl/cmd/istioctl/rbac.go

```diff
@@ -84,6 +86,37 @@ istioctl experimental rbac can -s source.namespace=foo POST rating /data -a dest
 	return cmd
 }
 
+func upgrade() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "upgrade <rbac-file1> <rbac-file2> ...",
```

Comment:
Maybe just call it `convert-binding`, similar to https://github.com/istio/istio/blob/8d82dbe6377f2ab8ea24a12b9ef0c0a1f7cd2f13/istioctl/cmd/istioctl/convert_ingress.go#L39

Reply:
This also upgrade/converts `ServiceRole`. User's policy files can have both `ServiceRole` and `ServiceRoleBinding` in it.